src/components/header/_header.scss

```diff
@@ -134,6 +134,9 @@
     &__grid-top {
       min-height: 36px;
     }
+    &-service-nav__link {
+      color: $color-white;
+    }
```

Comment:
I think the hover style for these links should match other white on colour links, same as the [promo banner](https://ons-design-system.netlify.app/components/promotional-banner/) and [announcement panel](https://ons-design-system.netlify.app/components/panel/examples/announcement/),  for example.